pkg/controller/datavolume/pvc-clone-controller.go

```diff
@@ -468,20 +468,32 @@ func (r *PvcCloneReconciler) isSourceReadyToClone(datavolume *cdiv1.DataVolume)
 
 // detectCloneSize obtains and assigns the original PVC's size when cloning using an empty storage value
 func (r *PvcCloneReconciler) detectCloneSize(syncState *dvSyncState) (bool, error) {
-	var targetSize int64
```

Comment:
special note on all the changes done in this function...was all done to get this crazy test to work:  https://github.com/kubevirt/containerized-data-importer/blob/35f1c79844b9aa76fccb6d4311669a6f8cf2e5e4/tests/cloner_test.go#L576